Resources/Prototypes/_White/Entities/Objects/Weapons/Guns/Turrets/ship_turrets.yml

```
@@ -0,0 +1,173 @@
+              min: 1
+              max: 1
+    - trigger:
+        !type:DamageTrigger
+        damage: 300
+      behaviors:
+        - !type:PlaySoundBehavior
+          sound:
+            collection: MetalGlassBreak
+    - trigger:
+        !type:DamageTrigger
+        damage: 150 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)